index.html

```diff
@@ -198,17 +198,33 @@ <h1 class="title">About Myself</h1>
                 contact me.
               </p>
               <p class="title_2">LET'S CONNECT</p>
-              <div class="items-about">
-                <img src="Images/Social/github.svg" class="github" alt=""/>
-                <img src="Images/Social/Linkedin.svg" class="linkedin" alt=""
-                />
-                <img src="Images/Social/mail.svg" class="mail" alt=""
-                />
-                <img src="Images/Social/other.svg" class="other" alt=""
-                />
-                <img src="Images/Social/twitter.svg" class="twitter" alt=""
-                />
-              </div>
+              <ul class="items-about">
+                <li>
+                  <a href=""
+                    ><img src="Images/Social/github.svg" class="github" alt=""
+                  /></a>
+                </li>
+                <li>
+                  <a href=""
+                    ><img src="Images/Social/Linkedin.svg" class="linkedin" alt=""
+                  /></a>
+                </li>
+                <li>
+                  <a href=""
+                    ><img src="Images/Social/mail.svg" class="mail" alt=""
+                  /></a>
+                </li>
+                <li>
+                  <a href=""
+                    ><img src="Images/Social/other.svg" class="other" alt=""
+                  /></a>
+                </li>
+                <li>
+                  <a href=""
+                    ><img src="Images/Social/twitter.svg" class="twitter" alt=""
+                  /></a>
+                </li>
+              </ul>
```

Comment:
- [ ] Images are still not being displayed properly.
- [OPTIONAL] These images are not necessarily a link. You could remove it from the `<a>` tags.
![image](https://user-images.githubusercontent.com/61919617/170838328-1b479e99-6354-4ffe-b0ec-e9fca5f52d43.png)